pkg/cert-refresh/cert_refresh.go

```diff
@@ -0,0 +1,215 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cert_refresh
+
+import (
+	"fmt"
+	"github.com/pingcap/tidb-operator/pkg/initializer"
+	"github.com/pingcap/tidb-operator/pkg/label"
+	certUtil "github.com/pingcap/tidb-operator/pkg/util"
+	batch "k8s.io/api/batch/v1"
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"time"
+)
+
+type RefreshManager struct {
+	kubecli kubernetes.Interface
+}
+
+type RefreshConfig struct {
+	Image               string
+	OwnerVersion        string
+	OwnerKind           string
+	OwnerName           string
+	OwnerUid            string
+	RefreshIntervalDays int
+}
+
+const (
+	configMapName  = "tidb-operator-initializer-config"
+	serviceAccount = "tidb-operator-initializer-sa"
+)
+
+var (
+	Days int
+)
+
+func NewRefreshManager(kubecli kubernetes.Interface) *RefreshManager {
+	return &RefreshManager{
+		kubecli: kubecli,
+	}
+}
+
+// RefreshManager list all the secrets created by Initializer and check whether it need refresh ca cert
+func (rm *RefreshManager) Run(podName, namespace string, config *RefreshConfig) error {
+	list, err := rm.checkCertsNeedRefresh(namespace, config.RefreshIntervalDays)
+	if err != nil {
+		return err
+	}
+	err = rm.setConfigOwnerReferences(podName, namespace, config)
```

Comment:
If `Refresh` find any cert need to be refreshed, it would create a job and set the job owner Reference as `Refresh` itself.